Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -12175,7 +12175,7 @@ a_Player:OpenWindow(Window);
 							Type = "number",
 						},
 					},
-					Notes = "Sets the skin part flags of the player.  The value should be a bitwise OR of several {{eSkinPart}} constants.",
+					Notes = "Sets the skin part flags of the player.  The value should be a bitwise OR of several {Globals|eSkinPart} constants.",
```

Comment:
This looks wrong. I think the best would be `{Globals#eSkinPart|eSkinPart}`, both here and above for the getter. That is, unless the APIDump code already strips the front part.